bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/repository/SharingStateRepository.kt

```diff
@@ -50,6 +50,6 @@ interface SharingStateRepository : PagingAndSortingRepository<SharingState, Long
             }
     }
 
-    fun findByExternalIdAndBusinessPartnerType(externalId: String, businessPartnerType: BusinessPartnerType): SharingState?
+    fun findByExternalId(externalId: String): SharingState?
```

Comment:
This works if we assume that the external IDs are unique over all business partner types. We planned to put some logic and migration in place to make sure that the external identifiers are really unique. However, I don't think that is the case yet. Is the pull request ready to be merged under these circumstances?

Reply:
Yeah, definitely still not ready to be merged yet, in regards to this. Forgot too put it in draft.

Reply:
Reverted for first query as unique constraints are not in place for now.